code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -62,3 +64,17 @@
 			for(var/mob/M in viewers(src))
 				M.show_message("<span class='notice'>\The [src] has been cut apart by [user] with \the [WC].</span>", 3, "You hear cutting.", 2)
 			qdel(src)
+			return
+		if(istype(W, /obj/item/weapon/storage/fancy/crayons)) /////addendum by flimflamm
+			var/decalselection = input("Please select a decal") in list( "cancel", "atmospherics", "bartender", "barber", "blueshield",	"brig_physician", "captain",
```

Comment:
You don't need a cancel option in the list. If you change your code from this:
`input() in list()`
to this:
`input() as null|anything in list()`
the input box will get a "Cancel" button, and pressing that will return null.